### PR TITLE
Fix Docstrings Formatting

### LINKED ISSUE #702: `ArrayImageSource` docstring formatting
It appears that the docstring for `ArrayImageSource` is improperly formatted as the parameters are not displaying correctly in the docs (see pic). Just need to add a blank line in the `__init__`.
 
Screen Shot 2022-09-26 at 9 17 59 AM

### PR DESCRIPTION
**Pull request by j-c-c**
Many docstrings had improper formatting. The two main issue were

1. missing blank line preceding `:param *:`
2. missing indent for line wraps

### CODE REVIEW

File: src/aspire/basis/fb_2d.py [hunk 1]
```diff
@@ -242,8 +242,8 @@ def _evaluate_t(self, v):
             must equal `self.sz`.
         :return: The evaluation of the coefficient array `v` in the dual basis
             of `basis`. This is an array of vectors whose last dimension equals
-             `self.count` and whose first dimensions correspond to
-             first dimensions of `v`.
+            `self.count` and whose first dimensions correspond to
```
Comment (garrettwrong) [inline src/aspire/basis/fb_2d.py:245]: Just curious, did this also cause trouble? (or just for consistency?)

**Review by garrettwrong — DISMISSED** [review]
Looks good.  Thanks for noticing this then going ahead and fixing it up.

So long as no conflicts should be good to merge. I think if any recent code regressed we can pick up in another small PR.

File: src/aspire/basis/fb_2d.py (hunk 1 above)
Comment (j-c-c) [reply src/aspire/basis/fb_2d.py:245]: This was just for consistency. Any amount of indentation will work, but a docstring linter would align all indented lines to the least indented. I've chosen to indent a tab (or 4 spaces).

File: src/aspire/source/image.py [hunk 2]
```diff
@@ -398,13 +398,18 @@ def cache(self):
     @property
     def images(self):
         """
+<<<<<<< HEAD
```
Comment (garrettwrong) [inline src/aspire/source/image.py]: think you left these in here :)

Comment (j-c-c) [reply src/aspire/source/image.py]: Aah! Thanks

**Review by janden — DISMISSED** [review]

**Comment by garrettwrong** [comment]
Fixed the minor conflict here, set to auto merge.

**Review by garrettwrong — APPROVED** [review]

**Pull request merged**